src/coreComponents/physicsSolvers/PhysicsSolverBase.hpp

```diff
@@ -693,6 +708,12 @@ class PhysicsSolverBase : public ExecutableGroup
 
     /// @return string for the allowNonConvergedLinearSolverSolution wrapper
     static constexpr char const * allowNonConvergedLinearSolverSolutionString() { return "allowNonConvergedLinearSolverSolution"; }
+
+    /// @return string for the writeLinearSystem wrapper
+    static constexpr char const * writeSolvingConvergenceCSVFlagString() { return "writeSolvingConvergenceCSV"; }
```

Comment:
```suggestion
    static constexpr char const * writeSolvingConvergenceCSVFlagString() { return "writeConvergenceCSV"; }
```
I wouldn't include "solver" as it is implied by where the field is defined.

Reply:
@arng40 can you rename `writeSolvingConvergenceCSVFlagString` consistently